app/views/groups/members/_current_user_is_member.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= link_to(
   t(:"groups.members.index.leave_group"),
-  group_member_path(id: member.id),
+  group_member_path(namespace, id: member.id),
```

Comment:
You shouldn't need to put `id: member.id` and can instead just replace the second argument with `member`. Same applies to `current_user_is_not_member.html.erb`

Reply:
Same will also apply to the same partials in project members

Reply:
20515370abc9a4fdf4d55d8a6b7a8fbe5a43aed9